docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -79,6 +79,10 @@ The following configuration properties are available:
      - The URL of the JSON Web Key Set (JWKS) endpoint on the authorization
        server. It provides Trino the set of keys containing the public key
        to verify any JSON Web Token (JWT) from the authorization server.
+   * - ``http-server.authentication.oauth2.userinfo-url``
+     - The URL of the IdPs ``/userinfo`` endpoint. If supplied then this URL is used
+       to validate the OAuth access token and retrieve any associated claims.
+       and retrieve associated claims.
```

Comment:
Mention that this is required if the Oauth server uses opague access tokens.

Reply:
👍